lib/elixir/src/elixir_def_overridable.erl

```diff
@@ -32,6 +32,13 @@ name(_Module, {Name, _} = Function, Overridable) ->
   {Count, _, _, _} = maps:get(Function, Overridable),
   elixir_utils:atom_concat([Name, " (overridable ", Count, ")"]).
 
+%% Returns the kind (def, defp, and so on) of the given overridable function.
+
+kind(Module, FunctionOrMacro) ->
```

Comment:
We could still calculate the args later on. The benefit of kind_and_name is just to avoid doing multiple lookups both in this module and in the translator. :)